docs/usage/docker.md

```diff
@@ -8,31 +8,35 @@ description: Docker Package Manager Support in Renovate
 Renovate supports upgrading dependencies in various types of Docker definition files:
 
 - Docker's `Dockerfile` files
-- Docker Compose files
+- Docker Compose `docker-compose.yml` files
 - CircleCI config files
-- Kubernetes manifests
+- Kubernetes manifest files
 - Ansible configuration files
 
 ## How It Works
 
-1.  Renovate will search each repository for any files matching each manager's configured `fileMatch` pattern(s)
-2.  Files that match the pattern(s) are parsed and checked to see if they contain any Docker image references (e.g. `FROM` lines in a `Dockerfile`)
-3.  If the image tag in use "looks" like a version (e.g. `node:12`, `node:12.1`, `node:12.1.0`, `node:12-onbuild`) then Renovate will look up the Docker registry to determine if any upgrades are available (e.g. `node:12.2.0`)
+1. Renovate searches in each repository for any files matching each manager's configured `fileMatch` pattern(s)
+1. Matching files are parsed, Renovate checks if the file(s) contain any Docker image references (e.g. `FROM` lines in a `Dockerfile`)
+1. If the image tag in use "looks" like a version (e.g. `node:14`, `node:14.1`, `node:14.1.0`, `node:14-onbuild`) then Renovate checks the Docker registry for upgrades (e.g. from `node:14.1.0` to `node:14.2.0`)
```

Comment:
I'm thinking of just using `node:10` or even `node:1` as example, so that it's easier to understand the examples.
It's all documentation anyways, and it's meant for the reader to get the idea of how it works.

What do you think? Do you want to use plausible Node versions, or do you rather have something like `node:10` or even `node:1`?